tensorflow/compiler/aot/BUILD

```diff
@@ -72,7 +72,13 @@ cc_library(
         "@llvm-project//llvm:Target",
         "@llvm-project//llvm:X86CodeGen",  # fixdeps: keep
         "//tensorflow/core:regexp_internal",
-    ] + if_llvm_aarch64_available([
+    ] + select({
+        "//tensorflow:linux_s390x": [
+            "@llvm-project//llvm:SystemZCodeGen",  # fixdeps: keep
+        ],
+        "//conditions:default": [
+        ],
```

Comment:
This needs to be in a macro, like `if_llvm_aarch64_available`.

Reply:
ok, I will put the "select" part in a macro under `tensorflow/core/platform/default/build_config.bzl` then use it here.